src/user-icon/user-icon.scss

```diff
@@ -67,8 +67,8 @@
     position: absolute;
     right: -2px;
     bottom: -2px;
-    width: calc(100% / 4);
-    height: calc(100% / 4);
+    width: calc(100% * 0.25);
```

Comment:
Maybe just `25%`? 😀 Same with `66.66...%`

Reply:
so there really is no need for calc.... just `width: 25%` right?

Reply:
Yeah. I would also go through all other instances of `calc` that you changed, to see if they can be simplified or undone (because using `/` in calc should not give you the deprecation warning).

Reply:
Turns out "all other instances" are all in this file 😄

Reply:
Updated to 25% and reverted others back to calc(100% / 1.5) since it was in calc function should be ignored by sass.